tasks/collect-collectors-data/README.md

```diff
@@ -0,0 +1,19 @@
+# collect-collectors-data
+
+Tekton task to collect the information related to collectors.
+
+The purpose of this task is to collect all the data and supply it to other tasks in the pipeline by creating
+json files for the different resources. This task is similar to collect-data but it only collects a subset of the
+resources. Specifically those related to collectors.
+
+A task result is returned for each resource with the relative path to the stored JSON for it in the workspace.
+
+## Parameters
+
+| Name                 | Description                                   | Optional | Default value |
+|----------------------|-----------------------------------------------|----------|---------------|
+| previousRelease      | Namespaced name of the previous Release       | No       | -             |
+| release              | Namespaced name of the Release                | No       | -             |
+| releasePlan          | Namespaced name of the ReleasePlan            | No       | -             |
+| releasePlanAdmission | Namespaced name of the ReleasePlanAdmission   | No       | -             |
+| subdirectory         | Subdirectory inside the workspace to be used. | Yes      | ""            |
```

Comment:
This being the only parameter with a period in the description is weird

Reply:
Removed. It needs to be removed from the collect-data task as well.